rust/noosphere-storage/src/db.rs

```diff
@@ -33,9 +33,15 @@ pub const BLOCK_STORE: &str = "blocks";
 pub const LINK_STORE: &str = "links";
 pub const VERSION_STORE: &str = "versions";
 pub const METADATA_STORE: &str = "metadata";
+pub const SCRATCH_STORE: &str = "scratch";
 
-pub const SPHERE_DB_STORE_NAMES: &[&str] =
-    &[BLOCK_STORE, LINK_STORE, VERSION_STORE, METADATA_STORE];
+pub const SPHERE_DB_STORE_NAMES: &[&str] = &[
```

Comment:
Strictly speaking I think this would call for a new version (and maybe migration, in `IndexedDB` terms). I don't think it's critical to do here, but something to think about. For example, anyone who used Sphere Viewer last Fall won't have a valid DB layout (for the given version) after this change.

Reply:
Looks like Rexie does some upgrade handling for e.g. adding object stores. Incrementing to version 2, and adding a quick test